### PR TITLE
Add basic View support - List the Views; List the tickets in a View

### PR DESCRIPTION
**Pull request by rcprcp**
Provide basic functionality for Views. Support getting a list of the
Views. Then, using the View id, we can get a list of the Tickets
in the View.

### CODE REVIEW

File: src/main/java/org/zendesk/client/v2/model/View.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class View implements Serializable {
+
+    private static final long serialVersionUID = 1L;
```
Comment (aheritier) [inline src/main/java/org/zendesk/client/v2/model/View.java:11]: It's probably better to generate a random number. (IDEs are generally proposing it)

**Review by aheritier — COMMENTED** [review]
Hi @rcprcp 

Thanks a lot for your contribution.
I think it's a good addition and I do not see any issue with the proposed code (just few minor comments).
The only missing part is to add some integration tests to avoid regressions but I know it's hard to write them without accessing to our sandbox (no thank you to Zendesk for not providing a shared environment for our community )
I you can review the 2 comments I pushed, I will try to write some tests on top of your PR ASAP

Cheers

File: pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
   <groupId>com.cloudbees.thirdparty</groupId>
   <artifactId>zendesk-java-client</artifactId>
-  <version>0.17.1-SNAPSHOT</version>
+  <version>0.17.2-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (rcprcp) [reply pom.xml]: replaced old value.

File: src/main/java/org/zendesk/client/v2/model/View.java (hunk 1 above)
Comment (rcprcp) [reply src/main/java/org/zendesk/client/v2/model/View.java]: fixed.

**Comment by rcprcp** [comment]
Hello @aheritier -  I have had a number of "git mishaps" with this change, but the most recent force-push should provide the changes you requested.   Additionally, I removed the "rawTitle" field from the View class. In my testing, the field was always empty.

Also, I have a simple test program for this PR - please review: 
https://github.com/rcprcp/ZendeskViewTest

Thanks!

**Comment by aheritier** [comment]
Perfect @rcprcp 
I will try to integrate it next week

**Comment by rcprcp** [comment]
Hi @aheritier  - wondering if you need anything from me to help move this along? 
Thank you.

**Comment by aheritier** [comment]
just lack of time @rcprcp 
@PierreBtz could you help to write some ITs. They should be very simple to do as far as I can see

**Comment by PierreBtz** [comment]
@rcprcp I took the liberty of pushing integration tests on your branch.

**Review by PierreBtz — APPROVED** [review]

**Comment by rcprcp** [comment]
@aheritier , @PierreBtz  - zendesk-java-client is an awesome library.  I appreciate all the time and effort you've invested in creating and maintaining it. And thank you very much for fixing this PR and emerging it.   I am glad that I could help.   :)

**Comment by PierreBtz** [comment]
@rcprcp thanks! Release 0.18.0 should be available shortly, I had a problem of expired gpg key, now I'm waiting for the gpg key servers to be synchronized and I hope to be able to complete the release process within the next hours...

**Pull request merged**